easybuild/easyconfigs/m/MotEvo/MotEvo-1.02-goolf-1.4.10.eb

```diff
@@ -0,0 +1,35 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock='MakeCp'
+
+name = 'MotEvo'
+version = '1.02'
+
+homepage = 'http://swissregulon.unibas.ch/fcgi/sr/software'
+description = """ MotEvo: a integrated suite of Bayesian probabilistic methods for 
+ the prediction of TFBSs and inference of regulatory motifs from multiple alignments 
+ of phylogenetically related DNA sequences """
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://www.swissregulon.unibas.ch/software/motevo/']
+sources = ['%(namelower)s_v%(version)s.tar.gz']
+
+start_dir = "source"
+
+parallel = 1
+
+# this requires glob support in MakeCp easyblock so
+# be sure to use at least EasyBuild-1.12 or newer
+easybuild_version = '1.12.0'
```

Comment:
I'd rather remove the above 3 lines since probability to be relevant by today is next to 0. (think: who sees this PR?)